attest/tpm.go

```diff
@@ -285,6 +285,25 @@ func intelEKURL(ekPub *rsa.PublicKey) string {
 	return intelEKCertServiceURL + url.QueryEscape(base64.URLEncoding.EncodeToString(pubHash.Sum(nil)))
 }
 
+const (
+	manufacturerAMD     = "AMD"
+	amdEKCertServiceURL = "https://ftpm.amd.com/pki/aia/"
+)
+
+func amdEKURL(ekPub *rsa.PublicKey) string {
```

Comment:
Can you provide more information on what the API expects the user to do to generate the request URL?

Is there documentation?